sql/src/test/java/io/crate/analyze/SelectStatementAnalyzerTest.java

```diff
@@ -161,15 +161,15 @@ public void testOrderedSelect() throws Exception {
     }
 
     @Test
-    public void testNegativeLiteral() throws Exception {
+    public void testNegativeLiteral() {
         AnalyzedRelation relation =  analyze("select * from sys.nodes where port['http'] = -400");
         Function whereClause = (Function) relation.where().query();
         Symbol symbol = whereClause.arguments().get(1);
-        assertThat(((Literal) symbol).value(), is(-400));
+        assertThat(((Literal<?>) symbol).value(), is(-400));
     }
 
     @Test
-    public void testSimpleSelect() throws Exception {
+    public void testSimpleSelect() {
```

Comment:
Please also revert these removals. It causes noise that is unrelated to the change of the PR.

 I know the IDE bugs you about it, but for tests it doesn't matter and it is usually annoying to re-add it once any of the methods tell the IDE that they throw something.

Reply:
My bad, I was getting blinded by the yellow column in the IDE, which hid some red.
I have seen people treat warnings as errors in other lives.